Source/Provers/LeanAuto/LeanAuto.cs

```diff
@@ -0,0 +1,1035 @@
+using System;
```

Comment:
Why not call the file LeanGenerator.cs?

Reply:
I renamed both the file and the class to use `LeanAutoGenerator`.